ngraph/core/src/ie/ie_layouts.cpp

```diff
@@ -0,0 +1,497 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
we can keep the location of these file in IE, but add to ngraph sources and exclude from IE ones.